### PR TITLE
CONCF-933 | Move getImage from CuratedContentSpecialController to CuratedContentController

### PR DESCRIPTION
**Pull request by vforge**
htps://wikia-inc.atlassian.net/browse/CONCF-933

/cc @Warkot @tomasznapieralski @bognix

### CODE REVIEW

**Comment by Warkot** [comment]
If that are all use cases, then LGTM. :+1:

File: extensions/wikia/CuratedContent/CuratedContentController.class.php [hunk 1]
```diff
@@ -523,6 +523,28 @@ public function getWikisWithCuratedContent() {
...
+			if ( !empty( $imageTitle ) && $imageTitle instanceof Title && $imageTitle->exists() ) {
+				$imageId = $imageTitle->getArticleID();
+			}
+		}
+
+		if ( !empty( $imageId ) ) {
+			$url = CuratedContentHelper::getImageUrl( $imageId );
+		}
+
+		$this->response->setVal( 'url', $url );
```
Comment (hakubo) [inline extensions/wikia/CuratedContent/CuratedContentController.class.php]: you could consider 
setValues as sugar here

**Comment by hakubo** [comment]
LGTM

**Comment by rogatty** [comment]
LGTM

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**